src/main/java/bhs/devilbotz/commands/auto/ScoreAndMobility.java

```diff
@@ -31,8 +31,11 @@ public ScoreAndMobility(Arm arm, DriveTrain drivetrain, double distance, Gripper
     addCommands(CommandDebug.start());
     addCommands(new AutoScore(arm, drivetrain, gripper));
     addCommands(new DriveStraightPID(drivetrain, -DriveConstants.POSITION_DRIVE_FROM_PORTAL));
+    addCommands(drivetrain.stop());
```

Comment:
This isn't a command?  How does this work?
Also might help to add some comments explaining what each step/cmd does.. and why.  There is no explanation what the starting pos of robot is.  What does AutoScore do? Why do we drive straight after? And then stop, etc.. Give a plain english explanation of what is supposed to happen in this auto routine.  Include start position, what happens, and end position.

Reply:
> This isn't a command? How does this work?
Hi @loafdog, `drivetrain.stop()` is indeed a command (`public Command stop()`), however, it does make sense to rename the method to `stopCommand()`. I'll make that change

> Also might help to add some comments explaining what each step/cmd does.. and why. 
There's already javadoc comments at the top of the file explaining what the command does.  There are javadocs for all sub commands, too....e.g. `AutoScore` also has comments at the top of the class.  Should I move the comments to be above the constructor so that it is displayed as a hover message in vscode?

> There is no explanation what the starting pos of robot is.
This class doesn't have a starting pos.  I believe the methods that do have a startAngle, it is documented.  Let me know if I missed a location where it isn't documented.